### DISCUSSION TITLE
posts/ruby-on-rails-prerequisites/

### DISCUSSION THREAD

**Discussion by giscus** [lukebewitched/lukebewitched.github.io#6] [Announcements]
# posts/ruby-on-rails-prerequisites/

Contains the necessary basics to get you up to speed

https://lukebewitched.github.io/posts/ruby-on-rails-prerequisites/

**Comment by lukebewitched** [comment]
Hello 👋,

This discussion platform is meant for the Kails Starting Page that contains the prerequisites for the Ruby On Rails comprehensive, beginner-friendly page. Feel free to ask, comment and suggest 👓💯.
